tilequeue/command.py

```diff
@@ -230,9 +259,11 @@ def make_tile_queue(queue_yaml_cfg, all_cfg, redis_client=None):
         queue_type = queue_yaml_cfg.get('type')
         assert queue_type, 'Missing queue type'
         if queue_type == 'sqs':
-            region = queue_yaml_cfg.get('region')
-            assert region, 'Missing queue region'
-            tile_queue = make_sqs_queue(queue_name, region)
+            sqs_cfg = queue_yaml_cfg.get('sqs', 'Missing queue sqs config')
```

Comment:
It looks like `queue_yaml_cfg` is a `dict` here, so would this assign the string `'Missing queue sqs config'` to `sqs_cfg` when the `'sqs'` entry is not present? It seems like that would give a confusing error message later. Would it be clearer to assert `sqs_cfg is not None`?